mountpoint-s3-client/src/checksums/crc32.rs

```diff
@@ -1,4 +1,4 @@
-use mountpoint_s3_crt_sys::aws_checksums_crc32;
```

Comment:
Another downside is that we go back to use the CRT crate directly from mountpoint, which I thought we would like to avoid (I know, log adapter).

Anyway, my last commit moves them back. We can always re-discuss later.